src/meta-srv/src/metasrv/builder.rs

```diff
@@ -217,6 +217,8 @@ impl MetaSrvBuilder {
             procedure_manager.clone(),
             kv_store.clone(),
             Arc::new(DatanodeClients::default()),
+            mailbox.clone(),
+            options.server_addr.clone(),
```

Comment:
Suggest to initialize the ddl_manager before the handler_group, because the initialization of other components is completed before the handler_group.